src/truncate/truncate.scss

```diff
@@ -24,4 +24,8 @@
   &.expandable:hover.css-truncate-target {
     max-width: 10000px !important;
   }
+
+  &.truncate-left {
```

Comment:
I think just like the `.expandable` modifier, this needs to be scoped to `.css-truncate-target`? It's for the case when `css-truncate truncate-left` gets added on a parent element? Multiline suggestion:

```diff
- &.truncate-left {
+ &.truncate-left .css-truncate-target,
+ &.truncate-left.css-truncate-target {
```

Reply:
Yes! Thanks for this good catch. I made this change in https://github.com/primer/css/pull/848/commits/7bd12a7e2697bc8fd96cd15baeb7e5b39fac2e9a.